### ISSUE TITLE
Section continuation to next page

### ISSUE THREAD

**Issue by arrhe5** [AmruthPillai/Reactive-Resume#967]
Assuming my work experience is beyond A4 size in that case we don't have a flexiblity to continue my text to next page, rather it kepp expanding to the same page.

**Describe the solution you'd like**
Rather expanding the current page it should give an option to continue in text page  and below section to adjust accordingly.

**Comment by kgotso** [comment]
It seems the layout does not force A4 Spec onto what is rendered. There is no easy way for one section to span more then one page. Not sure this makes sense as it should automatically figure out as it has the line breaks already as it currently just prints 1 page to pdf regardless of the line breaks

**Comment by arrhe5** [comment]
Yes. The Layout doesn't force to use A4 but _ would_ like to have it. Is there any way to achieve this? Section to span on consecutive  page.

**Comment by kgotso** [comment]
Agreed, have no idea as well. Looking at the code, hoping the Author can help

**Comment by AmruthPillai** [comment]
Basically, you would need to create a custom section (also call it Work Experience) and add the one work experience that's extra to that section. You can now move the new section to a new page.

This lets you manage your work experience in two pages.

**Comment by arrhe5** [comment]
Thank you.That's how. I'm doing it right now but wanted to know any other alternative exists.

**Comment by AthulMuralidhar** [comment]
this does not work / does not make sense - would be nice to see a big ass button that says move to page 2 or something like this - now it just looks plane ugly :/

**Comment by AmruthPillai** [comment]
> this does not work / does not make sense - would be nice to see a big ass button that says move to page 2 or something like this - now it just looks plane ugly :/

This does give me an idea about having a context menu option in each section, to move it to a new page. It's not going to be as intuitive as a big ass button, but it's definitely something :) Will look into this further this weekend.

Thank you!

**Comment by joesan** [comment]
Is there any update on this?

**Comment by AmruthPillai** [comment]
This issue was automatically closed as it was created before 2024-01-01.
